daemon/dns/dns.go

```diff
@@ -340,6 +334,16 @@ func unsetDNSWithResolvconf(iface string) error {
 }
 
 func setDNSDefault(addresses []string) error {
+	if internal.FileExists(resolvconfFilePath) {
+		info, err := os.Stat(resolvconfFilePath)
```

Comment:
We have an abstraction over filesystem stuff in fileshare. Could you extract it to some top level package(maybe `nordvpn-linux/internal` and use it here? Would be god for consistency.

Reply:
Do you mean creating a function which combines `internal.FileExists` and `os.Stat`? Because `os.Stat` is used in various places in the code and I don't see what would be a point in making its one-to-one copy in `internal`.

Reply:
maybe move the entire logic in a function e.g.`IsWritable` so it is easier to reuse in future?